code/game/turfs/open/floor/misc_floor.dm

```diff
@@ -204,8 +204,9 @@
 	desc = "Corrupted steel."
 	icon_state = "plating_rust"
 
-/turf/open/floor/plating/rust/plasma
-	initial_gas_mix = "plasma=104;TEMP=293.15"
+/turf/open/floor/plating/plasma/rust/New()
```

Comment:
Use `Initialize`, and change this to just add the component rather than replacing.